# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ash/x-cross.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Juck - let's build the build helpers using the system compiler.

  - Rene Rebe <rene@exactcode.de>

diff -ur ash-linux-0.2/Makefile ash-linux-0.2-fixed/Makefile
--- ash-linux-0.2/Makefile	1993-10-04 19:48:13.000000000 +0100
+++ ash-linux-0.2-fixed/Makefile	2005-02-02 11:02:26.198007168 +0100
@@ -40,19 +42,19 @@
 	touch init.c
 
 mkinit: mkinit.c
-	$(CC) $(CFLAGS) $(LDFLAGS) mkinit.c -o $@ $(LDADD)
+	$(BUILDCC) $(CFLAGS) $(LDFLAGS) mkinit.c -o $@ $(LDADD)
 
 nodes.c nodes.h: mknodes nodetypes nodes.c.pat
 	./mknodes nodetypes nodes.c.pat
 
 mknodes: mknodes.c
-	$(CC) $(CFLAGS) $(LDFLAGS) mknodes.c -o $@ $(LDADD)
+	$(BUILDCC) $(CFLAGS) $(LDFLAGS) mknodes.c -o $@ $(LDADD)
 
 syntax.c syntax.h: mksyntax
 	./mksyntax
 
 mksyntax: mksyntax.c parser.h
-	$(CC) $(CFLAGS) $(LDFLAGS) mksyntax.c -o $@ $(LDADD)
+	$(BUILDCC) $(CFLAGS) $(LDFLAGS) mksyntax.c -o $@ $(LDADD)
 
 clean:
 	rm -f core $(CLEANFILES) $(PROG) $(OBJS)
